log.go

```diff
@@ -30,6 +35,24 @@ func openLogWriter() *os.File {
 	return f
 }
 
+func rotateLog(path string) {
+	if _, err := os.Stat(path); err != nil && errors.Is(err, os.ErrNotExist) {
+		return
+	}
+
+	crashPath := rotatedLogPath(path)
+	err := os.Rename(path, crashPath)
+	if err != nil {
+		log.Println("Could not save crash file", crashPath)
```

Comment:
This log line is printed before the logfile is set to the new path, so this will go to stderr. Just confirming that is expected behavior? Would this be a case where we might want to use fyne.LogError instead?

Reply:
Yeah didn't want to write a new boot error to the old log

Reply:
I moved it to fyne.LogError on merge :)